cmd/minishift/cmd/profile/profile_set.go

```diff
@@ -38,6 +42,10 @@ func runProfile(cmd *cobra.Command, args []string) {
 	validateArgs(args)
 	profileName := args[0]
 
+	if profileName == profileActions.GetActiveProfile() {
+		atexit.ExitWithMessage(0, fmt.Sprintf(profileAlreadyActive, profileName))
```

Comment:
We don't want to exit here, just convey the msg to the user that profile is already active, because doing the command `minishift profile set` multiple times gives the ability to sync up the changes.

Reply:
@anjannath Right. @ulyssessouza As I mentioned in https://github.com/minishift/minishift/issues/1398#issuecomment-346935731 I would suggest not to change profile set for this issue.